slicedimage/_imagepartition.py

```diff
@@ -0,0 +1,29 @@
+from __future__ import absolute_import, division, print_function, unicode_literals
```

Comment:
I don't do too much 2.7 supported work -- is the future import needed in __init__?